lib/checks/color/color-contrast.json

```diff
@@ -4,7 +4,14 @@
 	"options": {
 		"noScroll": false,
 		"ignoreUnicode": true,
-		"ignoreLength": false
+		"ignoreLength": false,
+		"boldValue": 700,
+		"boldPt": 14,
+		"nonBoldPt": 18,
+		"minNormalContrastRatio": 0,
+		"normalContrastRatio": 4.5,
+		"minLargeContrastRatio": 0,
+		"largeContrastRatio": 3
```

Comment:
I think these values are a little complicated, I wonder if we couldn't break this down differently. How about if we have three checks:

```js
any: [
  'color-contrast-regular', // contrastRatio: 4.5
  'color-contrast-large', // contrastRatio: 3, minFontSize: 18
  'color-contrast-bold' // contrastRatio: 3, minFontSize: 14, minFontWeight: 700,
]
```